src/redux/categories/categoriesSlice.jsx

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories_arr: [],
+};
+
+const underConstructionReducer = () => (
+  <h3>Under Construction</h3>
+);
+
+// Includes an array of categories
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    underConstruction: underConstructionReducer,
```

Comment:
- [ ] According to the requirements:

![image](https://github.com/Zven94/Bookstore/assets/77069011/bfa8b49e-5683-439d-88e2-4f2e6b871e87)

Ensure that your reducer checks the status and always returns "Under construction". You could do something like this:

```javascript
    underConstruction: (state, action) => {
      state.categories_arr =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
```
Please consider fixing this! 🤓 🔧